gazebo/physics/Actor_TEST.cc

```diff
@@ -102,6 +102,62 @@ TEST_F(ActorTest, TrajectoryFromSDF)
   EXPECT_LT(fabs(actor->ScriptTime() - world->SimTime().Double()), 1.0 / 30);
 }
 
+//////////////////////////////////////////////////
+TEST_F(ActorTest, ActorCollision)
+{
+  // Load a world with an actor
+  this->Load("examples/plugins/actor_collisions/actor_collisions_noloop.world",
```

Comment:
The problem of loading this world in the test is that it assumes that the plugin is compiled and in the path. The test is usually ran in environments where this isn't true. For me, it printed this error and the test failed:

```
[Err] [Plugin.hh:178] Failed to load plugin libActorCollisionsPlugin.so: libActorCollisionsPlugin.so: cannot open shared object file: No such file or directory
```

I think we have 2 options:

1. Compile the plugin during the test. This can become tough, you'd need to do something [like this](https://github.com/osrf/gazebo/blob/gazebo11/test/examples/examples_build.cc), then add it to `GAZEBO_PLUGIN_PATH`, then be sure to cleanup at the end.
2. Change this test to just check the changes to `Actor.cc`. For example:
    * create a simple test world in [test/worlds](https://github.com/osrf/gazebo/tree/gazebo11/test/worlds) with false `loop` / `auto_start`
    * step until the animation ends
    * change the actor's pose from the test (to simulate what gravity would do in case the plugin had added collisions)
    * step a bit more, then check that the actor went back to the pose at the end of the loop

Reply:
Ok thank you, I would go for option 2 then.
So the test world will not include anymore the collision plugin, therefore I can remove the part of the test that checks if collisions are enabled.